examples/api/test/material/button_style/button_style.0_test.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/button_style/button_style.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets(
+      'Shows ElevatedButtons, FilledButtons, OutlinedButtons and TextButtons in enabled and disabled states',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(const example.ButtonApp());
+
+    expect(
+      find.byWidgetPredicate((Widget widget) {
+        return widget is ElevatedButton && widget.onPressed == null;
+      }),
+      findsOne,
+    );
+
+    expect(
+      find.byWidgetPredicate((Widget widget) {
+        return widget is ElevatedButton && widget.onPressed != null;
+      }),
+      findsOne,
+    );
+
+    //one OutlinedButton with onPressed null
```

Comment:
nit: end sentence with a period, here and elsewhere. Also capitalize beginning of sentence, here and elsewhere.